crates/router/src/core/user.rs

```diff
@@ -2510,12 +2504,13 @@ pub async fn switch_org_for_user(
             None,
             None,
             None,
+            Some(UserStatus::Active),
+            Some(1),
         )
         .await
         .change_context(UserErrors::InternalServerError)
         .attach_printable("Failed to list user roles by user_id and org_id")?
-        .into_iter()
-        .find(|role| role.status == UserStatus::Active)
+        .first()
```

Comment:
No `to_owned()` needed again.
```suggestion
        .pop()
```